### ISSUE TITLE
Feature: Allow overriding CSS through the gui  config

### ISSUE THREAD

**Issue by bearcatsandor** [Jas-SinghFSU/HyprPanel#431]
To facilitate more customization, it would be beneficial to allow users to override the css of modules and the bar itself though the config gui.  Something as simple as a text-area to allow them to enter their own css.

Of course knowledge of the existing classes would be necessary, but that could be added to the help page.

**Comment by rubiin** [comment]
Rather than here, this could be achieved via an alias if you are on a unix system.
There are few pros of not adding it here one is
1. You control the editor of your choice
2. Most editors will come with some sort of css intellisense by default which will also be needed for a working solution and  the css files are distributed across a number of files so a dedicated editor will help you more in getting idea of what file you are editing

**Comment by bearcatsandor** [comment]
Agreed.  Might instructions for that be added to the readme or a link to the instructions placed in the gui?

**Comment by rubiin** [comment]
Will update the docs.
